CHANGELOG.md

```diff
@@ -44,6 +44,7 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 - Converse.js UI is not used anymore, react components are used instead
 - Add anonymous_id parameter to register a user to a scheduled webinar
 - Update live layout for students
+- Display chat and viewers list in the tabs panel
```

Comment:
changelog is in the wrong commit IMO

Reply:
I agree with you, I was thinking it needed to be in the first commit of the feature.
Consequently, i changed where the changelog is. It is now in commit 'recycle(front) update tabs panel to also display viewers list', because this commit is the one which makes the feature appears and be used in the front.